src/documentation/gtfs_to_ntfs_specs.md

```diff
@@ -12,11 +12,12 @@ At the end of the conversion, a sanitizing operation is started on the final
 model. See [sanitizer.md](sanitizer.md) for more information.
 
 ### Prepending data
-The NTFS format introduce 2 objects to enable the manipulation of several datasets : contributors and datasets. Those two objects are not described here. The construction of NTFS objects IDs requires, for uniqueness purpose, that a unique prefix (specified for each source of data) needs to be included in every object's id.
-
+As explained in [common.md](common.md), a prefix is added to all identifiers during the conversion in order to guarantee uniqueness among objects IDs.
```

Comment:
I wonder if this should be part of all the other connectors too? Or do you plan to do that in future PRs?

Reply:
Yes, all the other connectors are to be updated at some point with this small section on data prefix.